starter-code/memory.js

```diff
@@ -33,6 +33,75 @@ var MemoryGame = function() {
     this.correctPairs = 0;
 };
 
+MemoryGame.prototype._shuffleCards = function() {
+  this.cards = _.shuffle(this.cards);
+};
+
+var previousCard;
+
+MemoryGame.prototype.selectCard = function(card) {
+  console.log(card.attr('name')+"selected");
+
+
+
+    if (this.selectedCards.length===1) {//If the user has selected a card in the last turn
+    this.selectedCards.push(card);
+    this.pairsClicked++;
+    $("#pairs_clicked").text(this.pairsClicked);
+    //console.log(this.selectedCards[0]+" "+this.selectedCards[1]);
+    // update the DOM
+    if (card.attr('name')===previousCard.attr('name')) { //Compare the newly selected card to the previously selected card. Are they of the same type?
```

Comment:
This works nicely, but notice that you could just access this.selectedCards[0] to fetch the previous cards :)